src/state/slices/portfolioSlice/selectors.ts

```diff
@@ -114,7 +114,7 @@ export const selectPortfolioAssetBalances = createDeepEqualOutputSelector(
     Object.values(accountBalancesById).reduce<Record<AssetId, string>>((acc, byAccountId) => {
       Object.entries(byAccountId).forEach(
         ([assetId, balance]) =>
-          (acc[assetId] = bnOrZero(acc[assetId]).plus(bnOrZero(balance).toString()).toString()),
+          (acc[assetId] = bnOrZero(acc[assetId]).plus(bnOrZero(balance)).toFixed()),
```

Comment:
we should probably use `toFixed` everywhere over `toString` - `toString` turns to scientific notation and loses precision